examples/skipfields/main.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2024 Greptime Team
```

Comment:
Do you think we should need an example for this feature? It's a very small feature and we can share one example for multiple features.